### ISSUE TITLE
Ability to pre-configure and save encounters

### ISSUE THREAD

**Issue by Mazzmnza42** [artg3-dev/DM_Tools#56]
**Is your feature request related to a problem? Please describe.**
I know you plan to create the ability to save Monster and NPC parties, but saving pre-configured encounters may also come in handy if you include Events. Ref #55

**Comment by artg3-dev** [comment]
So this is a party of monsters + events?

**Comment by Mazzmnza42** [comment]
Yes. I only included this because a added the event suggestion. But saving encounters wouldn't replace monster parties. I think it's important to be able to have monster / NPC parties separately. Then, you can add parties to an encounter, add events, and then adjust at which round each aspect of the encounter would start. you could then save all of that into an encounter. This way, you can have these ready to go before you start your session. You might not ever use that encounter again, but there's certainly a possibility you will use that party of monsters again in a different situation. That's why I think they should be separate.

**Comment by artg3-dev** [comment]
Ok that makes sense. I was thinking doing away with monster parties and having encounters be "monster parties"
